### ISSUE TITLE
Can no longer import Swagger files

### ISSUE THREAD

**Issue by cviebrock** [a85/POSTMan-Chrome-Extension#2303]
I get the following error when trying to import a swagger.json file (that I've previously been able to import):

```
Failed to import data: Could not import: TypeError: postmanConverters.swagger2_0Converter is not a constructor
```

Checking the console logs, I see:

```
Data is not YAML.
```

I tried converting my JSON file to YAML and importing it.  Same error message in the app, but no details in the console log.

Has Swagger import been removed?

---
1. Postman Version: 4.6.3
2. App (Chrome app or Mac app): Mac app
3. OS details: OS X 10.11.6

**Comment by cviebrock** [comment]
For fun, I tried pasting the raw text of my YAML file in the importer.  The console gave the same error, but with a bit more detail:

```
Uncaught TypeError: postmanConverters.swagger2_0Converter is not a constructor    requester.js:261321
    importData                              @ requester.js:261321
    handleImport                            @ requester.js:224925
    handleImport                            @ requester.js:225007
    handleClick                             @ requester.js:50562
    invokeGuardedCallback                   @ requester.js:4241
    executeDispatch                         @ requester.js:4086
    executeDispatchesInOrder                @ requester.js:4109
    executeDispatchesAndRelease             @ requester.js:3541
    executeDispatchesAndReleaseTopLevel     @ requester.js:3552
    forEachAccumulated                      @ requester.js:4383
    processEventQueue                       @ requester.js:3757
    runEventQueueInBatch                    @ requester.js:4412
    handleTopLevel                          @ requester.js:4428
    handleTopLevelWithoutPath               @ requester.js:14602
    handleTopLevelImpl                      @ requester.js:14582
    perform                                 @ requester.js:6572
    batchedUpdates                          @ requester.js:10596
    batchedUpdates                          @ requester.js:6080
    dispatchEvent                           @ requester.js:14713
```

Hope that helps.

**Comment by vegetableman** [comment]
hey @cviebrock we are looking in to the issue and will have a fix out for it this week. Thanks for taking time to report it 👍 .

**Comment by iamthemovie** [comment]
Any update on this? :)

**Comment by vegetableman** [comment]
@iamthemovie  The fix will be out early next week. Stay tuned for updates !.

**Comment by Iodine-** [comment]
looking forward to update as well 👍

**Comment by talalibdah1** [comment]
This happened only after the last update of postman. Same is happening when I try to import Runscope tests as well

**Comment by SamvelRaja** [comment]
@cviebrock @Iodine- @iamthemovie You can find the fix in our latest `4.7.0` update.
